engines/scumm/debugger.cpp

```diff
@@ -175,6 +193,180 @@ bool ScummDebugger::Cmd_IMuse(int argc, const char **argv) {
 	return true;
 }
 
+#if defined(ENABLE_SCUMM_7_8)
+bool ScummDebugger::Cmd_DiMuse(int argc, const char **argv) {
+	if (!_vm->_imuseDigital || _vm->_imuseDigital->isEngineDisabled()) {
+		debugPrintf("No Digital iMUSE engine is active.\n");
+		return true;
+	}
+
+	if (argc > 1) {
+		if (!strcmp(argv[1], "panic")) {
```

Comment:
maybe some better name instead "panic"?

Reply:
"panic" appears to be a fairly common command name in the audio software domain to denote "stop everything" commands (e.g. MIDI panic).
I can ditch that command and instead make people use "stop all" instead of creating another alias as "stopAll".

Reply:
please do